test/e2e/framework/node/wait.go

```diff
@@ -150,7 +150,7 @@ func WaitForNodeToBeReady(c clientset.Interface, name string, timeout time.Durat
 
 // CheckReady waits up to timeout for cluster to has desired size and
 // there is no not-ready nodes in it. By cluster size we mean number of Nodes
-// excluding Master Node.
+// excluding Control Plane Node.
```

Comment:
I think it's more appropriate to drop reference to master/control-plane entirely, e.g.

> By cluster size we mean number of schedulable Nodes